plugin-build/src/test/kotlin/io/sentry/android/gradle/SentryPluginTest.kt

```diff
@@ -186,6 +186,36 @@ class SentryPluginTest(
         assertTrue(":app:transformReleaseClassesWithAsm" in build.output)
     }
 
+    @Test
+    fun `applies only DB instrumentables when only DATABASE feature enabled`() {
```

Comment:
`m`: Should we also add a similar test for `when only FILE_IO enabled`?

Reply:
👍